README.md

```diff
@@ -58,6 +58,29 @@ To build Mixxx, run
 There should now be a `mixxx` executable in the current directory that you can
 run. Alternatively, can generate a package using `cpack`.
 
+### Using Dev Container
+
+> [!NOTE]
+> Dev container has recently been introduced and it is likely incomplete! Currently, it's been tested with `devpod` on Zed and Codium exclusively
```

Comment:
This will be outdated soon. Please change it that it is valid for long.

Reply:
Did you gain more experience with this meanwhile? 
This note is very prominent compared to the rest of the Readme. Can we remove it?